app/ocr.py

```diff
@@ -129,12 +131,38 @@ def get_country_of_origin(self) -> Union[str, None]:
 
     def get_date(self) -> str:
         """
-        • Returns date of the document. Easy to validate by the PDF filename.
+        • Finds all dates within document in all different formats
+        • Returns most recent date found within document
+        • Returns all dates in standard XX/XX/XXXX format 

```

Comment:
Good description of the function.
Relating to line 136, does this return all dates or just one date?

Reply:
The final output is one date. It initially pulls all of the dates before sorting through and finding the most recent since the different type cases have the correct date in different spots. Sorry I should have made that more clear!